src/otx/algorithms/segmentation/configs/ham_segnext_b/selfsl/hparam.yaml

```diff
@@ -0,0 +1,17 @@
+# Hyperparameters.
+hyper_parameters:
+  parameter_overrides:
+    learning_parameters:
+      batch_size:
+        default_value: 32
+      learning_rate:
+        default_value: 0.0001
+      learning_rate_warmup_iters:
+        default_value: 0
+      num_iters:
+        default_value: 10
```

Comment:
Are 10 epochs sufficient to obtain well pretrained weights?

Reply:
I think '10 epochs' is not sufficient actually to optimize pre-trained weights.
But the objective of Self-SL (warmstart) is to pre-train the model on the target dataset for a short time and expect fast model convergence with similar performance. (of course outperforming is the best :D)
When uploading the target dataset, pertaining is automatically started and it must be completed before the user completes the first labeling (8~12 images).
For SegNext-b, pretraining was taken about 50 min (generating pseudo masks : 16 min / pretraining : 33 min) with 4555 images for 10 epochs.
We can simply imagine eta for different #images (such as #100 : 21 + 43 = 64 s / #500 : 105 + 217 = 322 s).
I think 10 epochs can be the baseline time to be recommended and the users can adjust configurations if they want to improve pre-trained weights.